package.json

```diff
@@ -23,6 +23,7 @@
   },
```

Comment:
If you remove the prebuilds you should remove the others as well.

Reply:
What others  do you mean?

Reply:
I saw that you removed the other OS prebuilds from the prebuilds folder, so I was thinking there is probably some in ```lib/prebuilds``` as well.

Reply:
No there is no prebuilds files in `lib` directory!